### DISCUSSION TITLE
When will the new mybatis-generator version be released

### DISCUSSION THREAD

**Discussion by svenmathijssen** [mybatis/generator#964] [General]
See the comment in https://github.com/mybatis/generator/issues/917: _In the next version of the generator, the default will switch to use the "jakarta" namespace instead of "javax"._

When will this 'next version' be released? We now have to artificially import the old javax.annotation-api (alongside jakarta inject api).

**Comment by jeffgbutler** [comment]
Released today - February 19, 2023
